src/Storages/MergeTree/MergeTreeDataMergerMutator.h

```diff
@@ -193,7 +193,7 @@ class MergeTreeDataMergerMutator
 
 
     /// The approximate amount of disk space needed for merge or mutation. With a surplus.
-    static size_t estimateNeededDiskSpace(const MergeTreeData::DataPartsVector & source_parts);
+    static size_t estimateNeededDiskSpace(const MergeTreeData::DataPartsVector & source_parts, const bool & is_merge);
```

Comment:
we need this interface to stay compatible with any other parts which are or can use it
```suggestion
    static size_t estimateNeededDiskSpace(const MergeTreeData::DataPartsVector & source_parts, const bool & use_estimation = false);
```